lib/wallet/src/Cardano/Wallet/Read/Tx/Certificates.hs

```diff
@@ -1,68 +0,0 @@
-{-# LANGUAGE DataKinds #-}
-{-# LANGUAGE FlexibleContexts #-}
-{-# LANGUAGE StandaloneDeriving #-}
-{-# LANGUAGE TypeApplications #-}
-{-# LANGUAGE TypeFamilies #-}
-{-# LANGUAGE UndecidableInstances #-}
-
--- |
--- Copyright: © 2020-2022 IOHK
--- License: Apache-2.0
---
--- Raw certificate data extraction from 'Tx'
---
-
-module Cardano.Wallet.Read.Tx.Certificates
```

Comment:
Oh — was the deletion of this module here intentional? 🤔

Reply:
no idea what it was doing there :shrug:

Reply:
truth is that it was in good company, unneeded stuff for the future ... my bad. I put that back ..